exporters/otlp/otlpmetric/internal/transform/metricdata_test.go

```diff
@@ -365,7 +365,7 @@ func TestTransformations(t *testing.T) {
 	require.Equal(t, pbScopeMetrics, sm)
 
 	// Resource Metrics.
-	rm, err := ResourceMetrics(otelResourceMetrics)
+	rm, err := ResourceMetrics(&otelResourceMetrics)
```

Comment:
I suggest changing the `otelResourceMetrics` variable initialization code (and add `&` there)

Reply:
updated as suggested